lib/msgServer/msgStream/transports/http-polling/index.js

```diff
@@ -202,18 +207,17 @@ HttpPollingHost.prototype.respondToHead = function () {
 
 
 HttpPollingHost.prototype._closeConnection = function () {
-	if (!this.res) {
+	if (this.res.finished) {
 		return;
 	}
 
-	this.res = null;
```

Comment:
I would keep this *and* the finished test. If we try to write twice in the same tick, I'm not sure if `finished` would always be set to the value we would hope it to be.